Starting evaluation of experiment COF-NASA-MSL-M-2-507aaea1141c95e8718e71d965e42aa9-1
=============================================

Performing execution for UNSUPERVISED algorithm COF
Running container 'ghcr.io/timeeval/cof:0.3.0' with env='{'LOCAL_UID': '501', 'LOCAL_GID': '501'}' in ExecutionType.EXECUTE mode.
Restricting container to 1.0 CPUs and 15.000 GB RAM

#### Docker container logs ####
useradd warning: user's uid 501 outside of the UID_MIN 1000 and UID_MAX 60000 range.
Config: AlgorithmArgs(customParameters=CustomParameters(n_neighbors=50, random_state=42), dataInput='/data/M-2.test.csv', dataOutput='/results/docker-algorithm-scores.csv', executionType='execute', modelInput='/results/model.pkl', modelOutput='/results/model.pkl')
Traceback (most recent call last):
  File "/app/algorithm.py", line 69, in <module>
    main(config)
  File "/app/algorithm.py", line 49, in main
    clf = COF(
  File "/usr/local/lib/python3.8/site-packages/pyod/models/cof.py", line 83, in __init__
    super(COF, self).__init__(contamination=contamination)
  File "/usr/local/lib/python3.8/site-packages/pyod/models/base.py", line 74, in __init__
    raise ValueError("contamination must be in (0, 0.5], "
ValueError: contamination must be in (0, 0.5], got: 0.501098

###############################

Docker algorithm failed with status code '1', consider container logs above.
